packages/contracts-bedrock/deploy-config/goerli.json

```diff
@@ -1,26 +1,39 @@
 {
-  "l1StartingBlockTag": "0xafce66a0a2446856112e4069b275ad32b1f4a607888f9c4c59eddf9be81f8670",
+  "finalSystemOwner": "DUMMY",
+  "controller": "DUMMY",
+
+  "l1StartingBlockTag": "DUMMY",
   "l1ChainID": 5,
-  "l2ChainID": 111,
+  "l2ChainID": 420,
   "l2BlockTime": 2,
 
-  "maxSequencerDrift": 1000,
-  "sequencerWindowSize": 120,
+  "maxSequencerDrift": 3600,
+  "sequencerWindowSize": 7200,
   "channelTimeout": 120,
-  "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
-  "batchInboxAddress": "0xff00000000000000000000000000000000000002",
-  "batchSenderAddress": "0xa11d2b908470e17923fff184d48269bebbd9b2a5",
-
-  "systemConfigOwner": "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720",
 
-  "l2OutputOracleSubmissionInterval": 6,
+  "p2pSequencerAddress": "DUMMY",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000000",
+  "batchSenderAddress": "DUMMY",
+  "l2OutputOracleSubmissionInterval": 20,
   "l2OutputOracleStartingTimestamp": -1,
-  "l2OutputOracleProposer": "0x6c23a0dcdfc44b7a57bed148de598895e398d984",
-  "l2OutputOracleChallenger": "0x6925b8704ff96dee942623d6fb5e946ef5884b63",
+  "l2OutputOracleProposer": "DUMMY",
+  "l2OutputOracleChallenger": "DUMMY",
+  "finalizationPeriodSeconds": 2,
+
+  "systemConfigOwner": "DUMMY",
+
+  "l2GenesisBlockGasLimit": "0x1C9C380",
```

Comment:
We set the limit rather than the target here. With an elasticity of 10 a `30MM` limit implies at `3MM` target which is `1.5 mgas/sec`. I thought we wanted `1.25 mgas/sec` which is a limit of `25MM`, but a `30MM` limit is also fine.

Reply:
Which value (`25MM` or `30MM`) do you think we should start with? Not married to either.

Reply:
25MM. That's been the standard so far